README.md

```diff
@@ -136,7 +136,8 @@ class CloneOptions {
 }
 
 int Main(string[] args) {
-  return CommandLine.Parser.Default.ParseArguments<AddOptions, CommitOptions, CloneOptions>(args)
```

Comment:
Ah, sorry, ignore that. It was an extension method. Okay, I will clean that up, thanks for the suggestion.